README.md

```diff
@@ -1,16 +1,248 @@
-# Title
+# Methods in Ruby
 
 ## Learning Goals
 
--SWBAT 1
--SWBAT 2
+- Describe routines and procedures for our code using methods
```

Comment:
This is the first lesson in this block. Is this language approp. for someone who's never built a code before?